doc/plugin_server_nodeattestor_aws_iid.md

```diff
@@ -81,10 +81,14 @@ This plugin generates the following selectors related to the instance where the
 
 | Selector            | Example                                               | Description                                                      |
 |---------------------|-------------------------------------------------------|------------------------------------------------------------------|
+| Availability Zone   | `aws_iid:az:us-west-2b`                               | The Availability Zone in which the instance is running.          |
+| IAM role            | `aws_iid:iamrole:arn:aws:iam::123456789012:role/Blog` | An IAM role within the instance profile for the instance         |
+| Image ID            | `aws_iid:image:id:ami-5fb8c835`                       | The ID of the AMI used to launch the instance.                   |
+| Instance ID         | `aws_iid:instance:id:ami-5fb8c835`                    | The ID of the instance.                                          |
```

Comment:
AWS Instance IDs follow a format of `i-[a-f0-9]{17}`, e.g. `i-0b22a22eec53b9321`

Can we update the example to reflect this format?

Reply:
Sure, i wrongly copied the image id 🙈

Reply:
[e35da0a](https://github.com/spiffe/spire/pull/3640/commits/e35da0a54f3d955f7f072c64980dfcccf60edf72)